spec/models/entry_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+
+RSpec.describe Entry, type: :model do
+  let(:basket) { create(:basket) }
+  let(:imported) { build(:imported, basket: basket) }
+  let(:exempt) { build(:exempt, basket: basket) }
+  let(:imported_exempt) { build(:importedExempt, basket: basket) }
```

Comment:
Just as Ruby convention, variables are written in lower case & underscore: `importedExempt` => `imported_exempt`